PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/mysql/MySQLPropertyView.java

```diff
@@ -154,7 +155,7 @@ private void onSaveButtonClicked(ActionEvent e) {
             }
             boolean allowAccessToLocal = connectionSecurity.getAllowAccessFromLocalMachineCheckBox().getModel().isSelected();
             if (!originalAllowAccessToLocal.equals(allowAccessToLocal)) {
-                MySQLMvpModel.FirewallRuleMvpModel.updateAllowAccessToLocalMachine(subscriptionId, property.getServer(), allowAccessToLocal);
+                AzureMySQLService.FirewallRuleService.getInstance().updateAllowAccessToLocalMachine(subscriptionId, property.getServer(), allowAccessToLocal);
                 originalAllowAccessToLocal = allowAccessToLocal;
```

Comment:
you have to check the result if update really succeeds.

Reply:
updated